eth/bloom_test.go

```diff
@@ -0,0 +1,54 @@
1d4e878d2f69d9af5"],"transactionHash":"0x0b45136ab4c9e7bb5663f3f3bb20dfed41405b0f537aa0f1f2d0d3484fc0e5ee","transactionIndex":"0x6c"},{"address":"0xb66dce2da6afaaa98f2013446dbcb0f4b0ab2842","blockHash":"0x4c77ccf7965128f95692ba17d5144754db9f79f925e1d3a9b214b102cb67c574","blockNumber":"0x979f18","data":"0x0000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000001477686f69732e666f725f73616c652e76616c7565000000000000000000000000","logIndex":"0xaa","removed":false,"topics":["0x7ae4f661958fbecc2f77be6b0eb280d2a6f604b29e1e7221c82b9da0c4af7f86","0xea4333cc4318e442d02983339923154dda48a758dfad7f11d4e878d2f69d9af5","0xa9815c025ce12e710afe7e2fe25a449b3f41a183573c781fb33a0911e8ebf302"],"transactionHash":"0x0b45136ab4c9e7bb5663f3f3bb20dfed41405b0f537aa0f1f2d0d3484fc0e5ee","transactionIndex":"0x6c"},{"address":"0xb66dce2da6afaaa98f2013446dbcb0f4b0ab2842","blockHash":"0x4c77ccf7965128f95692ba17d5144754db9f79f925e1d3a9b214b102cb67c574","blockNumber":"0x979f18","data":"0x00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000001477686f69732e666f725f73616c652e76616c756500000000000000000000000000000000000000000000000000000000000000000000000000000000000000047472756500000000000000000000000000000000000000000000000000000000","logIndex":"0xab","removed":false,"topics":["0x851ffe8e74d5015261dba0a1f9e1b0e5d42c5af5d2ad1908fee897c7d80a0d92","0xea4333cc4318e442d02983339923154dda48a758dfad7f11d4e878d2f69d9af5","0xa9815c025ce12e710afe7e2fe25a449b3f41a183573c781fb33a0911e8ebf302","0x6273151f959616268004b58dbb21e5c851b7b8d04498b4aabee12291d22fc034"],"transactionHash":"0x0b45136ab4c9e7bb5663f3f3bb20dfed41405b0f537aa0f1f2d0d3484fc0e5ee","transactionIndex":"0x6c"},{"address":"0x604652c6a6e606669bdccde15519644b869a97a9","blockHash":"0x4c77ccf7965128f95692ba17d5144754db9f79f925e1d3a9b214b102cb67c574","blockNumber":"0x979f18","data":"0x000000000000000000000000000000000000000000000000000000005ea02069","logIndex":"0xac","removed":false,"topics":["0x708861da4aa6afa7c59ec1b6595bfacef3b852515c06faafba359c70bc640e8c","0x000000000000000000000000d6a491ab3cf8b574d070ea5e131a8be3139c4a68"],"transactionHash":"0x391336bd4d6d3ba95292817c78c8e0f2625554558db85e66a3d7749caeb4193d","transactionIndex":"0x6f"},{"address":"0xdac17f958d2ee523a2206206994597c13d831ec7","blockHash":"0x4c77ccf7965128f95692ba17d5144754db9f79f925e1d3a9b214b102cb67c574","blockNumber":"0x979f18","data":"0x0000000000000000000000000000000000000000000000000000000326736a80","logIndex":"0xad","removed":false,"topics":["0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef","0x0000000000000000000000007effadd894fdd14d147cebe733e269967e85efa0","0x000000000000000000000000c98c11289ed310d23f2dcd7bd879bc280f02c4d3"],"transactionHash":"0xa6c4ce14d0c7b13f46de26a577ff5c754f4c6c976f7b7f827c89013fa8291465","transactionIndex":"0x70"}]`)
```

Comment:
LOL github just _gives up_ on rendering this line.  you can see it in "raw" mode though: https://raw.githubusercontent.com/INFURA/go-ethlibs/7cadac68154282af8092274e3637e94f470d773e/eth/bloom_test.go